backendAS/seedSongs.js

```diff
@@ -0,0 +1,130 @@
+// seed
+// define headers
+
+const faker = require('faker');
+const moment = require("moment");
+const fs = require('fs');
+
+const { Writeable, Readable } = require('stream');
+
+class ReadSongs extends Readable {
+  constructor(options) {
+    super(options);
+    this.goal = 10 * 1000 * 1000;
+    this.rowsgenerated = 0;
+    this.progressBarLength = 25;
+    this.startTime = moment();
+  }
+
+  _read() {
+    if (this.goal <= this.rowsgenerated + 1) {
+      this.push(null);
+    } else {
+      let container = '';
+      for (let i = 0; i < 1000; i++) {
+        const songname = faker.lorem.sentence(2); // string
+        const artistname = faker.name.findName(); // String
+        const imgurl = faker.image.imageUrl(); // string
+        const hashtag = faker.random.word();
+        const timeelapsed = faker.date.past();
+        const start = 0;
+        const songlength = faker.random.number({ min: 120, max: 620 });
+        const decibel = faker.random.number({ min: 62, max: 80 });
+        const songurl = 'songurl'; // aws string link
+        container += `${this.rowsgenerated},${songname},${artistname},${imgurl},${hashtag},${timeelapsed},${start},${songlength},${decibel},${songurl}/n`;
+        this.rowsgenerated++;
+      }
+      this.push(container);
+      this._logPercentComplete();
+
+    }
+  }
+
+  _logPercentComplete() {
```

Comment:
Is the percent logging function intended to only be used while generating songs? Comment generation lacks this functionality.

Reply:
I will add this.